packages/checkbox-group/test/checkbox-group.test.js

```diff
@@ -448,7 +448,7 @@ describe('vaadin-checkbox-group', () => {
     let error, helper, label;
 
     beforeEach(() => {
-      group = fixtureSync('<vaadin-checkbox-group helper-text="Choose one"></vaadin-checkbox-group>');
+      group = fixtureSync('<vaadin-checkbox-group helper-text="Choose one" label="Label"></vaadin-checkbox-group>');
```

Comment:
Previously, the field groups used to add the label id to the `aria-labelledby` attribute even when no label is provided. 

The PR aligns it with non-group fields so that the label id is now only added to the attribute when a label is present.